lua/lv-which-key/init.lua

```diff
@@ -191,6 +152,18 @@ local mappings = {
   },
 }
 
+local vopts = {
+  mode = "v", -- NORMAL mode
+  prefix = "<leader>",
+  buffer = nil, -- Global mappings. Specify a buffer number for buffer local mappings
+  silent = true, -- use `silent` when creating keymaps
+  noremap = true, -- use `noremap` when creating keymaps
+  nowait = false, -- use `nowait` when creating keymaps
+}
+local vmappings ={
```

Comment:
Please add a note above this which says:
-- NOTE: Prefer using : over <cmd> in the command expression as the later avoids going back in normal-mode.
-- see https://neovim.io/doc/user/map.html#:map-cmd

Reply:
Alright. Added that